src/main/java/de/hhu/bsinfo/dxram/commands/Start.java

```diff
@@ -36,6 +38,24 @@
     @CommandLine.Option(names = "--level", description = "The log level to use")
     private String m_logLevel = "INFO";
 
+    @CommandLine.Option(
+            names = { "--storage", "--kvsize", "--kv" },
+            description = "Amount of main memory to use for the key value store in MB.")
+    private int m_storage = 128;
+
+    @CommandLine.Option(
+            names = "--handler",
+            description = "Number of threads to spawn for handling incoming and assembled network messages.")
+    private int m_handler = 2;
+
+    @CommandLine.Option(
+            names = { "--master-slave-role", "--msrole" },
+            description = "Compute to assign to the current instance (master, slave or none).")
```

Comment:
```suggestion
            description = "Compute role to assign to the current instance (${COMPLETION-CANDIDATES}).")
```

Small typo here and picocli offers the option to [print out all possible values for enum types](https://picocli.info/#_enum_types).

Reply:
Great hint, done! However I observed the following behavior:
description = 

> Compute role to assign to the current instance (master, slave or none).

but only UpperCase _NONE, MASTER, SLAVE_ works! Is there an option for case insensitivity?

Reply:
Thanks! Yes, in fact there is [setCaseInsensitiveEnumValuesAllowed](https://picocli.info/apidocs/picocli/CommandLine.html#setCaseInsensitiveEnumValuesAllowed-boolean-) to enable this behaviour. This could be added within the [main](https://github.com/hhu-bsinfo/dxram/blob/development/src/main/java/de/hhu/bsinfo/dxram/Runner.java) method.

Reply:
Done!

Reply:
Excellent, thank you!